packages/test-utils/src/wrapper.js

```
@@ -582,3 +594,15 @@ export default class Wrapper implements BaseWrapper {
     this.element.dispatchEvent(event)
   }
 }
+
+export const enableAutoDestroy = (hook: Function) => {
+  if (Wrapper._instances) {
+    throwError('enableAutoDestroy cannot be called more than once')
+  }
+
+  Wrapper._instances = []
+
+  hook(() => {
+    ;(Wrapper._instances || []).forEach(wrapper => wrapper.destroy())
```

should we throw an error here if `Wrapper._instances` is not defined?